modules/eks/echo-server/remote-state.tf

```
@@ -6,3 +6,13 @@ module "eks" {
 
   context = module.this.context
 }
+
+module "alb-controller-ingress-group" {
```

Dashes shouldnt be used in resource or module names. Let's change to underscore.

```suggestion
module "alb_controller_ingress_group" {
```